design/one-pager-ignore-changes.md

```diff
@@ -0,0 +1,354 @@
+parameters during updates. Terraform addresses this use case with its
+[ignore_changes] functionality.
+
+An additional benefit of ignoring some fields would be that it would be
+possible to have different Crossplane Compositions managing a single
+external resource. This could open up some interesting new possibilities
+for Crossplane.
+
+The obvious way would be to just not set the field in the `spec.forProvider`.
+However, due to [Late Initialization], some fields are being set in
+`spec.forProvider` without any user input. Plus there are some cases
+where the field is required/wanted upon creation, but not on updates.
+
+So in this document, the aim will be to introduce a solution in Crossplane
+for ignoring some managed resource parameters during updates.
+
+## Goals
+
+* Design a functionality that allows for ignoring select fields during
+  resource updates.
+
+## Proposal
+
+Proposed solution is to use the new [managementPolicy] field which came with
+the [ObserveOnly] feature and add a new management policy that will 
+skip late initialization. The loss the information that the
+late initialization was providing would be offset by the `status.atProvider`
+which contains the resource state on the provider side and is being
+added alongside [ObserveOnly].
+
+In addition to the new management policy, we will also add a new field
+`spec.initProviders` which would contain parameters that should only be
+used at the creation time. This would allow users to specify fields that
+should be ignored on updates, but still be used during creation. This can
+be added as a separate feature.
+
+## Implementation
+
+### Management policy
+
+Proposed name for the new management policy is `PartialControl`, which
+would fit with the current `FullControl` and insinuate that the resource
+is not fully managed by Crossplane. Some other proposals for the naming
+are: `ExplicitControl`, `SelectiveControl`, `IgnoreLateInit`.
+
+The new management policy would be implemented in the `crossplane-runtime`
+and would just ignore the [ResourceLateInitialize] condition and 
+skip the managed resource update [here][Late Initialization Update] if
+the management policy is set to `PartialControl`.
+
+### Deletion policy
+
+`DeletionPolicy` was planned to be deprecated in favour of the new management
+policies according to [the ObserveOnly design doc.][ObserveOnly] However,
+with `PartialControl` it's not clear if it should delete or orphan the
+external resources. So to avoid creating `PartialControl` and
+`PartialControlOrphanOnDelete` management policies, we will keep the
+`DeletionPolicy` to be able to specify the behaviour. In that case, we will
+need to deprecate and remove the `OrphanOnDelete` management policy as it
+won't be needed anymore.
+
+So the management policies would be:
+- `FullControl`
+- `PartialControl`
+- `ObserveOnly`
+
+With a note that for `ObserveOnly` the `DeletionPolicy` will be ignored,
+as we are in read-only mode.
```

Comment:
This seems implicit and surprising. I'd prefer we honor deletion policy even in observe-only mode. In practice I would expect almost everyone to set `deletionPolicy: Orphan` when they set `managementPolicy: ObserveOnly`.

Reply:
Removed the part about ignoring the `deletionPolicy` My worry was that the default `deletionPolicy` is `Delete` so users who do not set the `deletionPolicy: Orphan` could be surprised. But probably less intuitive it would be if it was ignored altogether. 

Not sure how to improve this experience except noting it very clearly in the documentation.